configs/segcore.yaml

```diff
@@ -0,0 +1,25 @@
+* GLOBAL:
```

Comment:
why is segcore not follow style of milvus.yaml?

Reply:
this yaml is for easylogging, easylogging has special request for yaml format.